torchdrivesim/behavior/common.py

```diff
@@ -1,2 +1,5 @@
-class InitializationFailedError(RuntimeError):
+from invertedai.error import InvertedAIError
+
+
+class InitializationFailedError(InvertedAIError):
```

Comment:
If we inherit from `InvertedAIError`, I would move this to `torchdrivesim.behavior.iai`.